cmake/templates/local_setup.bash.in

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+# generated from catkin/cmake/templates/local_setup.bash.in
+
+CATKIN_SHELL=bash
+
+# source setup.sh from same directory as this file
+_CATKIN_SETUP_DIR=$(builtin cd "`dirname "${BASH_SOURCE[0]}"`" > /dev/null && pwd)
+. "$_CATKIN_SETUP_DIR/setup.sh" --extend --local
```

Comment:
I would suggest adding `--local` to the supported command line options in the `setup.sh` template documentation: https://github.com/ros/catkin/blob/7fcfe2ce33b68e83ff74990aa87ed0fb84b87983/cmake/templates/setup.sh.in#L6-L9

Reply:
Good point: added in 802bf106ec66990d573c0bfd6c2801c75770c1c7.